openhabian-setup.sh

```diff
@@ -1439,6 +1441,8 @@ amanda_setup() {
             fi
       done
   fi
+  
+  /usr/sbin/usermod -a -G backup openhabian
   /usr/sbin/chpasswd <<< "${backupuser}:${password}"
   /usr/bin/chsh -s /bin/bash ${backupuser}
```

Comment:
You didn't change that last line but could you please explain the need for it?

Reply:
which line?
chsh ? User needs to have a proper shell.
usermod? User can use default openhabian user to see Amanda config, browse storage

Reply:
I mean the chsh line. Why exactly does a non-interactive system account need a shell? That's normally not needed or if it is, other solutions are available.

Reply:
You need to switch to the backup user in case of restore, so while you normally wouldn't use it interactively, in this case you need to.

Reply:
I see. I don't particularly like that we are exposing yet another user+password combination via SSH. Wdyt?

Reply:
Well, Amanda requires this, so little chance to work around that without breaking anything.
If you're truely worried, you can add a 'DenyUsers backup' line to /etc/sshd/sshd_config.

Reply:
Well, wether I am worried or not is not the point. The system should still be secure for the end user. For now I'd be happy if you simply mentioned that "the password has to be safe because it will be able to log in remotely" in the enter password dialog.